app/models/data/user_module_completion.rb

```diff
@@ -29,7 +29,7 @@ def dashboard
       # @param mod_name [String]
       # @return [Integer]
       def module_count(mod_name)
-        User.all.count { |user| user.module_completed?(mod_name) }
+        User.registration_complete.count { |user| user.module_completed?(mod_name) }
```

Comment:
Does the spec generate a user who hasn't completed registration, or is there a spec for the scope already?

Reply:
It does now [f38d173](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/749/commits/f38d1738264e7d3996999db663972aef998d1e54)